src/main/java/graphql/schema/SchemaTransformer.java

```diff
@@ -34,6 +34,55 @@
 @PublicApi
 public class SchemaTransformer {
 
+    /**
+     * Different ways to traverse a schema object during transformation.
+     *
+     * ALL - traverse all top level types (query, mutations, additional types etc)
```

Comment:
I would put the javadoc for each enum ON the enum

Mind you you have javadoc, which is more than a certain German does so.....

Reply:
Updated.